Gems/PhysX/Code/Source/Joint/PhysXJoint.cpp

```diff
@@ -133,6 +133,7 @@ namespace PhysX
         : PhysXJoint(sceneHandle, parentBodyHandle, childBodyHandle)
     {
         m_pxJoint = Utils::PxJointFactories::CreatePxPrismaticJoint(configuration, sceneHandle, parentBodyHandle, childBodyHandle);
+        AZ_Printf("PhysXPrismaticJoint::PhysXPrismaticJoint", " PhysXPrismaticJoint %d", m_pxJoint.get());
```

Comment:
update: Might want to add a reminder to remove this before moving out of draft status

Reply:
Removed.